pkg/graph/resolvers/mto.resolvers.go

```diff
@@ -34,7 +33,7 @@ func (r *modelsToOperationMatrixResolver) Milestones(ctx context.Context, obj *m
 }
 
 // Solutions is the resolver for the solutions field.
-func (r *modelsToOperationMatrixResolver) Solutions(ctx context.Context, obj *models.ModelsToOperationMatrix) ([]*model.MTOSolution, error) {
+func (r *modelsToOperationMatrixResolver) Solutions(ctx context.Context, obj *models.ModelsToOperationMatrix) ([]*models.MTOSolution, error) {
 	panic(fmt.Errorf("not implemented: Solutions - solutions"))
```

Comment:
@OddTomBrooks , I think this is where the dataloader for get by model plan id would be called. I think the get by milestone id can / should hold off until we link solutions and milestones.